.ideainspect

```diff
@@ -21,3 +15,10 @@ resultdir: target/inspection-results
 
 # IDEA project root directory containing the ".idea" directory
 # rootdir: .
+
+# Library folder containing idea.properties More info at:
+# http://tools.android.com/tech-docs/configuration
+# lib: ~/Library/Preferences/AndroidStudio2.1/
+
+# Custom IDEA Scope name
```

Comment:
It is the name in the XML file, because one xml file can have multiple scope profiles. IDEA automatically detects them when booted up, so we pass the name of the one we want.